source/best-practices/file-organization.rst

```diff
@@ -0,0 +1,158 @@
+文件管理最佳实践
+================
+
+:本节贡献者: `田冬冬 <https://me.seisman.info>`__ （作者）、`姚家园 <https://github.com/core-man>`__ （作者）
+:最近更新日期: 2021-01-12
+:预计阅读时间: X 分钟
+
+正确地管理自己电脑中的文件是非常重要的。错误的文件管理方式会使得你花费更多的时间在寻找文件上，
+拖慢你的科研进度，还可能导致文件的大量冗余与误删。
+这一节，我们总结了自己在科研工作中关于文件组织与管理的经验，供读者参考。
+
+本文主要面向 Linux 和 macOS 用户，下面提到的\ **家目录**\ 在 Linux 下指的是 :file:`/home/用户名`，
+在 macOS 下指的是 :file:`/Users/用户名`。
+
+文件管理基本原则
+----------------
+
+本文的文件组织与管理遵循如下几条基本原则：
+
+1. 保持家目录干净整洁
+2. 所有科研相关的文档、代码、图片等均放在家目录的子目录下
+3. 家目录下的每个子目录都有特定的用途，用于放置特定的文件
+
+目录结构
+--------
+
+根据以上原则，我们推荐在家目录下建立如下子目录：
+
+-   Linux 或 macOS 系统在安装后自动创建的目录
+
+    - :file:`Documents`
+    - :file:`Downloads`
+    - :file:`Videos`
+    - :file:`Music`
+    - :file:`Desktop`
+
+-   为了满足日常科研需求，在家目录下新建的、用于存储科研相关文件的目录
+
+    - :file:`bin`
+    - :file:`src`
+    - :file:`software`
+    - :file:`opt`
+    - :file:`projects`
+    - :file:`codes`
+    - :file:`data`
+    - :file:`workspace`
+
+-   根据需求创建其他目录
+
+子目录用途
+----------
+
+根据原则 3，家目录下的每个子目录应有特定的用途。下面将详细阐述每个子目录的用途，
+以及哪些文件应该放在哪个子目录下。
+
+software
+^^^^^^^^
+
+该目录用于放置从网络上下载的软件包（如 SAC、GMT、TauP 等）的原始压缩包，
+相当于一个备份。若压缩包数目较少，则所有软件压缩包都放在该目录下；
+若压缩包数目很多，可以考虑根据软件的用途对软件进行分类，放在该目录的不同子目录下。
+同时建议在该目录下建一个 README 文件，记录软件的名称、用途以及官方下载地址。
+
+src
+^^^
+
+该目录用于放置别人提供的科研相关的软件源码，每个软件对应 :file:`src` 目录下的
+一个子目录。比如 fk、gCAP 等的源码都放在这里。
+
+codes
+^^^^^
+
+该目录用于放置自己写的具有\ **通用性**\ 的软件包。这些软件包可以实现某类特定的
+功能，且不局限于某一个特定的科研项目。
+
+opt
+^^^
+
+我们推荐将地震学相关软件安装到家目录的 :file:`opt` 目录下，而不要安装到
+:file:`/opt/` 或 :file:`/usr/local` 下。
+
+该目录下可放置两类科研相关软件：
+
+1. 无需编译即可直接使用的软件，如 SAC、TauP、SOD 等
+2. 需要编译且指定安装位置的软件，如 GMT
+
+bin
+^^^
+
+家目录下的 :file:`bin` 目录，用于存放一些简单的可执行文件以及脚本。
+将该目录的路径加入到环境变量 **PATH** 中，则可以在终端或脚本中直接调用
+这些可执行文件与脚本。
+
+决定是否要将某个可执行文件或脚本放在该目录下，可以基于如下一些原则：
+
+1. 若某个软件包只提供了单个可执行文件，则可放在该目录下
```

Comment:
I cannot remember a common-used soft example. rdseed is not used any more.

Reply:
I hava `cpdf` in my bin directory, but that's not for research use.

Reply:
Okay. we may add it if we find one.